src/test/java/ch/interlis/iox_j/wkb/Iox2wkbTest.java

```diff
@@ -152,6 +155,16 @@ public void surfaceZToExtendedWkb() throws Exception {
         String ewkbText=Base64.encodeBytes(ewkb);
         assertEquals("AQoAAIABAAAAAQkAAIABAAAAAQIAAIAEAAAAAAAAAABMHUEAAAAAABfxQAAAAAAAAPA/AAAAAICEHkEAAAAAAIjzQAAAAAAAAPA/AAAAAODIIEEAAAAAAPn1QAAAAAAAAPA/AAAAAABMHUEAAAAAABfxQAAAAAAAAPA/",ewkbText);
     }
+
+    @Test
+    public void surfaceWithArcToWkb() throws Exception {
+        IomObject multiSurface=getSurfaceWithArc();
+        Iox2wkb convEwkb=new Iox2wkb(2,java.nio.ByteOrder.LITTLE_ENDIAN, true);
+        byte ewkb[]=convEwkb.surface2wkb(multiSurface, true, 0.001, false);
+        // verify
+        String ewkbText=Base64.encodeBytes(ewkb);
+        assertEquals("AQoAAAABAAAAAQkAAAADAAAAAQIAAAACAAAAAAAAgI5NREEAAAAA5BEzQQAAAICOTURBAAAAAOwEM0EBCAAAAAUAAAAAAACAjk1EQQAAAADsBDNBAAAAgC9TREEAAAAA7AQzQQAAAICCUkRBAAAAACESM0EAAACATlNEQQAAAAD0FDNBAAAAALBPREEAAAAAmBczQQECAAAAAgAAAAAAAACwT0RBAAAAAJgXM0EAAACAjk1EQQAAAADkETNB",ewkbText);
```

Comment:
assert mit WKT ergänzen

Reply:
Der WKBReader von JTS kann keine CURVEPOLYGON lesen.

Ich schlage for anstatt WKT den Hex-String des WKB zu asserten. Umgewandelt mit [Iox2wkb.bytesToHex](https://github.com/claeis/iox-ili/blob/fb639168f15c40c049b1cea77ddf23a7286f6dcb/src/main/java/ch/interlis/iox_j/wkb/Iox2wkb.java#L97). So können alle Tests gleich assertet werden und der Hex-String ist einfacher zu interpretieren als der Base64-String.